docker-compose.override.yml

```diff
@@ -27,6 +27,10 @@ services:
     volumes:
       - ./gitea:/go/src/code.gitea.io/gitea
 
+  postgres:
+    ports:
+      - '5432:5432'
```

Comment:
Why do we need this?

Reply:
It's handy for using `psql` while developing. We expose a bunch of other ports for development by the way.

Reply:
OK!

FYI, I use this line `docker exec -it  <container-name/id> psql -U <postgres-username>` to get a postgres REPL inside a _running_ container.